### DISCUSSION TITLE
Add examples

### DISCUSSION THREAD

**Discussion by drokeye** [HuyaneMatsu/hata#44] [Ideas]
Any plan about adding examples for core concepts which would be helpful to get started with the library?

**Comment by albertopoljak** [comment]
Do these help? https://github.com/HuyaneMatsu/hata/tree/master/docs
There is intro which is sort of like a tutorial and there are topics which contain detailed explanations&examples for specific things.

**Reply by HuyaneMatsu** [reply]
[Done](https://github.com/HuyaneMatsu/hata/tree/master/docs/examples).

**Comment by lrnzdc3696** [comment]
> hata needs an examples folder

**Reply by lrnzdc3696** [reply]
Someone said/suggested on discord <(￣︶￣)>

**Reply by asleep-cult** [reply]
for the love of god shut up and go to bed.

**Reply by lrnzdc3696** [reply]
Ok and ? ¯\\_( ͡° ͜ʖ ͡°)_/¯

**Comment by HuyaneMatsu** [comment]
Closing.
